addons/binding/org.openhab.binding.logreader/OSGI-INF/org.openhab.binding.logreader.internal.LogReaderHandlerFactory.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As DS annotations are used, *.xml should be added to .gitignore in that folder.

Reply:
Done